### ISSUE TITLE
Nesting menus forces the child menu to be "inline"

### ISSUE THREAD

**Issue by XuluWarrior** [grommet/grommet#1575]
In the example below because 'child' is nested in 'parent' the inline property is overriden with the value "explode" and so it is impossible make 'child' collapsed until clicked on.

I guess there's some usability reason for this but it would be nice to make the behaviour configurable.

```
<Menu id="parent">
  <Menu id="child" inline={false}>
  </Menu>
</Menu
```

**Comment by RyanCCollins** [comment]
Hey @XuluWarrior, thank you very much for posting this issue! 

Is it possible that you could post a codepen exhibiting this issue?  It will help us to better triage it.  When you open an issue on the grommet repo, we ask that you use the issue template (found [here](https://github.com/grommet/grommet/blob/master/.github/ISSUE_TEMPLATE.md)).  You can find the Grommet Hello World template [here](https://codepen.io/grommet/pen/gaEGPY).

**Comment by RyanCCollins** [comment]
Hey @XuluWarrior, any updates on this?

**Comment by karatechops** [comment]
@RyanCCollins @XuluWarrior I'm going to close this as it's been two weeks without follow up. Feel free to re-open this with some more context/ a Codepen is available.

**Comment by XuluWarrior** [comment]
I don't have access to a computer at the moment and probably won't for a while.

I am not going to try and write code on my phone but the code snippet I included above pasted into any example would demonstrate the issue. (Well you'd have to add a child element to the child menu to see the menu being displayed inline) 

Please note that this isn't a bug report. The behaviour is as currently designed. The code deliberately overrides the inline property of the child menu to "expanded". This used to be "exploded" but was fixed for https://github.com/grommet/grommet/issues/1574

(As a side note the tests that @alansouzati made to fix #1574 would demonstrate the behaviour I am referring to) 

I raised this issue to discuss what the reasoning behind overriding the developer's intentions were. And whether there was scope for change to allow developers to set inline to be false.

**Comment by blyefd** [comment]
@karatechops:  Here is a codepen which demonstrates the issue.  I would also like to have nested menus (not inline).
[https://codepen.io/anon/pen/EooJaK](https://codepen.io/anon/pen/EooJaK)
Can you please reopen is issue?
